cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -37,6 +37,8 @@ func appendTargetError(res *worker.OSBuildJobResult, err error) {
 }
 
 func (impl *OSBuildJobImpl) Run(job worker.Job) error {
+	errorLog := log.New(os.Stderr, "", 0)
+	errorLog.SetFlags(log.Ldate | log.Ltime)
```

Comment:
is this really needed? The standard logger is defined in this way:

```golang
const (
	LstdFlags     = Ldate | Ltime // initial values for the standard logger
)
var std = New(os.Stderr, "", LstdFlags)
```

Looks like we are instantiating the exact same thing.

See https://github.com/golang/go/blob/7ba8e796c91eaf4befcacc4d24127ae54475d6a5/src/log/log.go#L317

Reply:
Oh! Indeed! Imma revert this. Thanks.